Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -52,6 +54,141 @@ header: <wtf/text/AtomString.h>
+template: struct WebKit::TapIdentifierType
+template: struct WebKit::TextCheckerRequestType
+template: struct WebKit::UserContentControllerIdentifierType
+template: struct WebKit::WebExtensionContextIdentifierType
+template: struct WebKit::WebExtensionFrameIdentifierType
+template: struct WebKit::WebExtensionPortChannelIdentifierType
+template: struct WebKit::WebExtensionTabIdentifierType
+template: struct WebKit::WebExtensionWindowIdentifierType
+template: struct WebKit::WebPageProxyIdentifierType
+template: struct WebKit::WebTransportSessionIdentifierType
+template: struct WebKit::WebTransportStreamIdentifierType
+[WebKitPlatform, AdditionalEncoder=StreamConnectionEncoder] class WTF::ObjectIdentifier {
+    uint64_t toUInt64()
+}
+
+header: <wtf/ObjectIdentifier.h>
+template: class WebCore::ResourceLoader
+template: class WebCore::WebSocketChannel
+template: class WebCore::WebSocketChannelClient
+template: class WebCore::WebSocketChannelInspector
+template: enum class IPC::ConnectionSyncRequestIDType
+template: enum class JSC::MicrotaskIdentifierType
+template: enum class WebCore::BroadcastChannelIdentifierType
+template: enum class WebCore::DOMCacheIdentifierType
+template: enum class WebCore::DisplayConfigurationClientIdentifierType
+template: enum class WebCore::FileSystemHandleIdentifierType
+template: enum class WebCore::FileSystemSyncAccessHandleIdentifierType
+template: enum class WebCore::LibWebRTCSocketIdentifierType
+template: enum class WebCore::MainThreadPermissionObserverIdentifierType
+template: enum class WebCore::OpaqueOriginIdentifierType
+template: enum class WebCore::PortIdentifierType
+template: enum class WebCore::RTCDataChannelLocalIdentifierType
+template: enum class WebCore::RTCRtpScriptTransformerIdentifierType
+template: enum class WebCore::RenderingResourceIdentifierType
+template: enum class WebCore::ServiceWorkerIdentifierType
+template: enum class WebCore::ServiceWorkerJobIdentifierType
+template: enum class WebCore::ServiceWorkerRegistrationIdentifierType
+template: enum class WebCore::WebLockIdentifierType
+template: enum class WebCore::WebSocketIdentifierType
+template: enum class WebCore::WorkerFileSystemStorageConnectionCallbackIdentifierType
+template: enum class WebKit::GraphicsContextGLIdentifierType
+template: enum class WebKit::LegacyCustomProtocolIDType
+template: enum class WebKit::LibWebRTCResolverIdentifierType
+template: enum class WebKit::QuotaIncreaseRequestIdentifierType
+template: enum class WebKit::RemoteVideoFrameIdentifierType
+template: enum class WebKit::VideoDecoderIdentifierType
+template: enum class WebKit::VideoEncoderIdentifierType
+template: struct IPC::AsyncReplyIDType
+template: struct WebCore::WebSocketFrame
+template: struct WebKit::ConnectionAsyncReplyHandler
+template: struct WebKit::StorageAreaIdentifierType
+[WebKitPlatform, CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WTF::AtomicObjectIdentifier {
```

Comment:
It would be nice if we could make a way for AdditionalEncoder to only apply to the ones that need it.